1122.py

```diff
@@ -0,0 +1,17 @@
+from collections import defaultdict
+from typing import List
+
+class Solution:
+    def relativeSortArray(self, arr1: List[int], arr2: List[int]) -> List[int]:
+        only_arr1 = sorted([e for e in arr1 if e not in arr2])
+        num_pos = defaultdict(lambda: 0)
```

Comment:
For common default values like an `int`, you can just say `defaultdict(int)`, as mentioned [here](https://github.com/woke7/leetcode-python/commit/3f000781c8b70971b26b988834ab84606fdff4f3#r48915408).